### DISCUSSION TITLE
Overleaf 模版更新审核及状态跟踪

### DISCUSSION THREAD

**Discussion by RizhongLin** [TJ-CSCCG/TongjiThesis#48] [Announcements]
为确保用户能够及时了解模板更新的状态和进程，我们特此设立本讨论区。由于每次对模板的更新都需要提交至 Overleaf 进行审核，而这一审核过程可能需要一定的时间，导致 Overleaf 上的模板版本相较于 GitHub 仓库中的版本存在一定的延迟。

通过本讨论区，我们将提供以下信息：

1. **提交审核的更新**：每当我们向 Overleaf 提交新的模板更新时，我们将在此区域通报，以便用户知晓最新提交的内容。
2. **审核状态通报**：审核过程中，任何关于审核状态的变更，包括审核开始和结束的通知，都会在此更新。
3. **使用建议**：在 Overleaf 的模板还未更新前，用户可根据讨论区提供的信息，决定是否从 GitHub 直接下载最新版本使用。

**Comment by RizhongLin** [comment]
### :mega: 更新通知

- **2024/05/13**: 向 Overleaf 提交了模板的更新，截止至 #45。
- **2024/05/15**: 审核通过，已经更新。

**Comment by RizhongLin** [comment]
### :mega: 更新通知

- **2025/09/18**: 向 Overleaf 提交了模板的更新，截止至 #61。
- **2025/09/19**: 审核通过，已经更新。

**Comment by admin** [comment]
### :mega: 更新通知

- **2026/03/07**: 向 Overleaf 提交了模板的更新，截止至 7c04ba124a4c2461bc9b54d6cfe0c5355651dae9。
- **2026/03/19**: 审核通过，已经更新。
